index.html

```diff
@@ -0,0 +1,156 @@
+                <li class="link-list-item">3</li>
+                <li class="link-list-item">4</li>
+              </ul>
+          </li>
+          <li class="project-list-item">item 4
+            <img>
+            <section class="feature1">
+              <header class="title-feature1">header</header>
+              <p class="content-feature1">text</p>
+              <ul class="link-list">
+                <li class="link-list-item">1</li>
+                <li class="link-list-item">2</li>
+                <li class="link-list-item">3</li>
+                <li class="link-list-item">4</li>
+              </ul>
+          </li>
+        </ul> 
+        <header class="project-list-title">header</header>
+        <p class="app-feature">text</p>
+        <ul class="app-list">
+          <li class="app-list-item">1</li>
+          <li class="app-list-item">2</li>
+          <li class="app-list-item">3</li>
+        </ul>
+        <p class="app-feature">text</p>
+        <ul class="app-list">
+          <li class="app-list-item">1</li>
+          <li class="app-list-item">2</li>
+          <li class="app-list-item">3</li>
+        </ul>
+      </div> 
+    </section>
+    <div class="site-wrapper">
+      <section class="fourth-section">
+        <header class="title-toughts">header</header>
+        <img>
+        <p class="about-thoughts">text</p>
+      </section>
+    </div>
+  <div>
+    <div class="site-wrapper">
+      <section class="fifth-section">
+        <header class="fifth-section-header">header</header>
+        <p class="fifth-section-text">text</p>
+        <p class="fifth-section-text">text</p>
+        <p class="fifth-section-text">text</p>
+      </section>
+    </div>
+  </div>
+  <div>
+    <div class="site-wrapper">
+      <section class="sixth-section">
+        <header class="sixth-section-header">header</header>
+        <ul class="skills">
+          <li class="skills-item">1</li>
+          <li class="skills-item">2</li>
+          <li class="skills-item">3</li>
+          <li class="skills-item">4</li>
+          <li class="skills-item">5</li>
+          <li class="skills-item">6</li>
+        </ul>
+        <ul class="skills">
+          <li class="skills-item">1</li>
+          <li class="skills-item">2</li>
+          <li class="skills-item">3</li>
+          <li class="skills-item">4</li>
+          <li class="skills-item">5</li>
+          <li class="skills-item">6</li>
+        </ul>
+        <ul class="skills">
+          <li class="skills-item">1</li>
+          <li class="skills-item">2</li>
+        </ul>
+      </section>
+    </div>
+  </div>
+  <div class="site-wrapper">
+    <section class="seventh-section">
+      <header class="seventh-section-header">header</header>
+      <ul class="for-more">
+        <li class="for-more-item">1</li>
+        <li class="for-more-item">2</li>
+        <li class="for-more-item">3</li> 
+      </ul>
+    </section>
+  </div>
+</div>
+  </main>
```

Comment:
Something is off with the indentation here.

Reply:
Done.